Adding links to README of slides/recordings

ddsjoberg opened the pull request:
**What changes are proposed in this pull request?**
Added links to the R/Pharma materials to the README

**Reference GitHub issue associated with pull request.** _e.g., 'closes #<issue number>'_
closes #338

@bzkrouse I didn't see a video of our r/pharma workshop. Do you know if it exists?

@bzkrouse @jtalboys is it obvious enough that those icons are links to materials/videos? Is there a better icon for the slides/website link?

![image](https://github.com/user-attachments/assets/2f7b2763-809f-4f89-affc-5a3988b7db1a)

--------------------------------------------------------------------------------

Pre-review Checklist (if item does not apply, mark is as complete)
- [ ] **All** GitHub Action workflows pass with a :white_check_mark:
- [ ] PR branch has pulled the most recent updates from master branch: `usethis::pr_merge_main()`
- [ ] If a bug was fixed, a unit test was added.
- [ ] Code coverage is suitable for any new functions/features (generally, 100% coverage for new code): `devtools::test_coverage()`
- [ ] Request a reviewer

Reviewer Checklist (if item does not apply, mark is as complete)

- [ ] If a bug was fixed, a unit test was added.
- [ ] Run `pkgdown::build_site()`. Check the R console for errors, and review the rendered website.
- [ ] Code coverage is suitable for any new functions/features: `devtools::test_coverage()`

When the branch is ready to be merged:
- [ ] Update `NEWS.md` with the changes from this pull request under the heading "`# cards (development version)`". If there is an issue associated with the pull request, reference it in parentheses at the end update (see `NEWS.md` for examples).
- [ ] **All** GitHub Action workflows pass with a :white_check_mark:
- [ ] Approve Pull Request
- [ ] Merge the PR. Please use "Squash and merge" or "Rebase and merge".

_Optional Reverse Dependency Checks_:

- Install `checked` with `pak::pak("Genentech/checked")` or `pak::pak("checked")`
- Check dev versions of `cardx`, `gtsummary`, and `tfrmt` which are in the `ddsjoberg` R Universe

  ```shell
  Rscript -e "options(checked.check_envvars = c(NOT_CRAN = TRUE)); checked::check_rev_deps(path = '.', repos = c('https://ddsjoberg.r-universe.dev', 'https://cloud.r-project.org'))"
  ```

- Check CRAN reverse dependencies but run tests skipped on CRAN

  ```shell
  Rscript -e "options(checked.check_envvars = c(NOT_CRAN = TRUE)); checked::check_rev_deps(path = '.', repos = 'https://cloud.r-project.org')"
  ```

- Check CRAN reverse dependencies in a CRAN-like environment

  ```shell
  Rscript -e "options(checked.check_envvars = c(NOT_CRAN = FALSE), checked.check_build_args = '--as-cran'); checked::check_rev_deps(path = '.', repos = 'https://cloud.r-project.org')"
  ```

bzkrouse reviewed (approved):
@ddsjoberg I don't see the workshop on youtube, and instead of the scroll I also like the "file" icon since its more documentation based. But I don't mind the scroll! I like the youtube icon as it is.

ddsjoberg commented:
> @ddsjoberg I don't see the workshop on youtube, and instead of the scroll I also like the "file" icon since its more documentation based. But I don't mind the scroll! I like the youtube icon as it is.

Thanks for reviewing @bzkrouse ! FYI I added the link to our PHUSE materials, because those will be largely ready by the time we make the release. The icon I wanted for the slides was the projector screen, but it's part of the PRO plan. But I found the display icon and it looks pretty similar, so I updated to that one. FYI
